SPIKE-NOTES.md

```diff
@@ -0,0 +1,107 @@
+# Spike notes
+
+In this spike pull request, I am showing how we can allow LB3 developers to
+expose their legacy applications in an LB4 project and thus allow them to run
+both old LB3 and new LB4 endpoints in the same process.
+
+The solution has two major parts:
```

Comment:
Are these going to be permanent part of LB4? Or are we getting rid of them after some time, when?

Reply:
`app.mountExpressRouter(basePath, router, spec)` will become a permanent part of `@loopback/rest`. I see it as a generic extension point that can be useful beyond LB3 migration.

`Lb3AppBooter` will live in its own package, we can decide how long we want to support it and when to deprecate it, independently on the rest of the framework.

Reply:
`app.mountExpressRouter` is a good addition for extensibility and customizability.